axelor-human-resource/src/main/resources/views/ProjectTask.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_7.2.xsd">
+
+  <form id="hr-project-task-form" name="project-task-form" title="Task"
+    model="com.axelor.apps.project.db.ProjectTask" extension="true">
+
+    <extend target="/">
+      <attribute name="onSave"
+        value="action-sprint-validate-upadte-or-create-project-plannings,action-project-task-method-update-project-plannings"/>
```

Comment:
Don't need to generate or update planification at save, just need it at sprint onchange in sprint-project-task-form